.gitignore

```diff
@@ -2,8 +2,38 @@ packwiz-installer-bootstrap.jar
 packwiz-installer.jar
 completion.sh
 reload.bat
-# Exclude exported CurseForge zip files
-*.zip
+config/sodium*
+config/iris*
+.fabric
+backups
+coremods
+crash-reports
+ESM
+fancymenu*
+!fancymenustuff
+item_renderer
+local
+logs
+saves
+screenshots
+ModTranslations
+observable_profiles
+rei_exports
+shaderpacks
```

Comment:
Things you want ignored locally but shouldn't be just ignored for everybody should just not be added to commits. Adding them to the gitignore will prevent any future changes in there until its removed from the gitignore.